iFixit/react-commerce
# Bump Cypress to version 10.9

Migrates our cypress test files to the new project structure for the latest version.

Updates the collection scroll test, which broke when we updated to the latest version. See 8d28e73c727274dbd3f10563b3532eae873fd233 for the workaround.

https://docs.cypress.io/guides/references/changelog#10-9-0
https://github.com/cypress-io/cypress/releases

## QA

Visit a collections page. Scroll down to the bottom, and click the next page, prev page, etc buttons. Every time you click one of those buttons, you should be scrolled to the top, and it should be a [smooth](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView#parameters) scroll.
